apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -302,12 +350,17 @@ private Config(
         int port,
         String inProcessServerName,
         Duration maxIdle,
-        String authenticationToken) {
+        @Nullable Duration minIdleTimeBeforeConnectionDrop,
+        @Nullable String authenticationToken) {
       this.serverType = Preconditions.checkNotNull(serverType, "serverType is required");
       this.host = host;
       this.port = port;
       this.inProcessServerName = inProcessServerName;
       this.maxIdle = maxIdle;
+      this.minIdleTimeBeforeConnectionDrop =
+          minIdleTimeBeforeConnectionDrop == null
+              ? Duration.ofMinutes(4)
```

Comment:
Can this use the default specified in AppConfiguration so we have it defined only in one place?

Reply:
Cross module circular dependencies like that are undesirable.  Also, the default defined at the application level is used when constructing instances of this class so this class level default is never used in the application.  I'll just change it to Long.MAX_VALUE so the default is just to not use this new feature unless a value is supplied.

Reply:
Default changed.